pkg/types/gcp/platform.go

```diff
@@ -123,8 +123,10 @@ type Platform struct {
 
 	// FirewallRulesManagement specifies the management policy for the cluster. Managed indicates that
 	// the firewall rules will be created and destroyed by the cluster. Unmanaged indicates that the
-	// user should create and destroy the firewall rules.
-	// +default="Managed"
+	// user should create and destroy the firewall rules. If firewall rules management value is unset,
+	// then the OpenShift Installer will determine if the user has the permissions to create
+	// and destroy firewall rules. If the service account has permissions to create and destroy firewall
+	// rules, then the value is set to "Managed" otherwise it will be set to "Unmanaged".
```

Comment:
This only applies to XPN install only. We should mention the behaviour for non-XPN...